docs/syntax-reference.rst

```diff
@@ -440,6 +440,28 @@ The WITH clause supports the following properties:
     :start-line: 2
     :end-line: 6
 
+.. _insert-into:
+
+INSERT INTO
+-----------
+
+**Synopsis**
+
+.. code:: sql
+
+    INSERT INTO stream_name
+      SELECT select_expr [., ...]
+      FROM from_item
+      [ WHERE condition ]
+      [ PARTITION BY column_name ];
+
+**Description**
+
+Stream the result of the SELECT query into an existing stream and its underlying topic.
```

Comment:
Shouldn't we clarify semantics such as ordering guarantees (or the lack thereof)?